packages/dev/core/src/scene.ts

```diff
@@ -750,34 +751,106 @@ export class Scene extends AbstractScene implements IAnimatable, IClipPlanesHold
     public _registeredForLateAnimationBindings = new SmartArrayNoDuplicate<any>(256);
 
     // Pointers
+    private _pointerPickingConfiguration = new PointerPickingConfiguration();

```

Comment:
So, I just wanted to understand this a little better.  Is this just supposed to be an object that holds all pointer picking specific options?  If so, is there any reason why we wouldn't want to make it public so that a user could create a configuration that could be shared with multiple scenes?

Reply:
It is not public to not change the backward compatible API
I did not want to add more fields to the scene object as we are too close from being slow down (https://www.eternalcoding.com/my-fat-objects-are-slower-than-my-thin-ones/)